buildtest/buildsystem/base.py

```diff
@@ -527,10 +540,12 @@ def _write_test(self, lines=None):
         return testpath
 
     def _get_test_lines(self):
-        """Given test metadata, get test lines to write to file or show.
-        """
+        """Given test metadata, get test lines to write to file or show."""
+
         # Every test starts with cd to TESTDIR
-        lines = ["cd $TESTDIR"]
+        lines = ["#!/bin/bash"]
```

Comment:
How does this honor the user choice if the choice is /bin/sh?

Reply:
so would you want the following
``shell: bash`` or ``shell: /bin/bash`` --> ``#!/bin/bash``
``shell: sh`` or ``shell: /bin/sh`` --> ``#!/bin/sh``. 

Is that right?